PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ar.cxx

```diff
@@ -71,314 +108,366 @@ static constexpr std::string_view TrackVariableNames[kLAST_TrackVariable] = {"Pt
                                                                              "TPCCrossedRows",
                                                                              "TPCChi2",
                                                                              "ITSClusters"};
-// prefixes
-static constexpr std::string_view ECrecoBefore = std::string_view("reco/EventControl/before/RB_");
-static constexpr std::string_view ECrecoAfter = std::string_view("reco/EventControl/after/RA_");
-static constexpr std::string_view ECsimBefore = std::string_view("sim/EventControl/before/SB_");
-static constexpr std::string_view ECsimAfter = std::string_view("sim/EventControl/after/SA_");
-static constexpr std::string_view TCrecoBefore = std::string_view("reco/TrackControl/before/RB_");
-static constexpr std::string_view TCrecoAfter = std::string_view("reco/TrackControl/after/RA_");
-static constexpr std::string_view TCsimBefore = std::string_view("sim/TrackControl/before/SB_");
-static constexpr std::string_view TCsimAfter = std::string_view("sim/TrackControl/after/SA_");
 
+// common info string for all configurables
 std::string info = std::string(": Hist bins, Hist lower edge, Hist upper edge, lower cut, upper cut, cut ON(1)/OFF(-1)");
 
-}; // namespace MultiParticleCorrelationsARTaskGlobalConfig
+const int MaxHarmonic = 10;
+const int MaxPower = 10;
 
-namespace AR = MultiParticleCorrelationsARTaskGlobalConfig;
+// function for computing the absolute distance of the primary vertex form the origin
+inline float abs(float vx, float vy, float vz)
+{
+  return std::sqrt(vx * vx + vy * vy * vz * vz);
+}
+
+// generice function for checking if the value of a variable passes a cut
+inline bool SurviveCut(std::vector<float> ConfigValue, float Value)
+{
+  bool flag = true;
+  // check if the cut is configured to be use in the first place
+  if (ConfigValue.at(kOCUT) > 0.) {
+    // check if the value of the variable is not lower than the lower bound and
+    // not not larger than the upper bound
+    if (!(Value > ConfigValue.at(kLCUT) && Value < ConfigValue.at(kUCUT))) {
+      flag = false;
+    }
+  }
+  return flag;
+}
+}; // namespace MultiParticleCorrelationsARTaskGlobalVariables
+
+// use an alias for our namespace
+namespace AR = MultiParticleCorrelationsARTaskGlobalVariables;
```

Comment:
```suggestion
using namespace AR = MultiParticleCorrelationsARTaskGlobalVariables;
```

Reply:
I acutally like using the namespace explicitly. Even though its more typing, I think it is easier to read and find the stuff during debugging.

Reply:
ah, you are right, I was thinking about `using` as creating an alias, but it does not work for namespaces in this way,